x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/columns/series_actions.tsx

```diff
@@ -182,7 +182,7 @@ const DELETE_SERIES_TOOLTIP_LABEL = i18n.translate(
 const VIEW_SAMPLE_DOCUMENTS_LABEL = i18n.translate(
   'xpack.observability.seriesEditor.sampleDocuments',
   {
-    defaultMessage: 'View sample documents',
+    defaultMessage: 'View transaction in Discovers',
```

Comment:
Should be "Discover" not "Discovers"

```suggestion
    defaultMessage: 'View transaction in Discover',
```